types/default-theme.d.ts

```diff
@@ -7,6 +7,11 @@ export namespace DefaultTheme {
      */
     logo?: string
 
+    /**
+     * Hide title when logo is avaiable
+     */
+    hideTitle?: boolean
```

Comment:
How about we make this `siteTitle?: string | false` and disable when it's `false`. This way, we can let user customize even the text as well.

- When it's `string` - Use it as nav title.
- When it's `false` - Do not display title.
- When iT's `undefined` - use `config.title` as title.